frontend/packages/client/src/components/ProposalsList/ProposalCardBody.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { stripHtml } from "string-strip-html";
+
+const ProposalCardBody = ({ name, body, cardWidth, inlineImage }) => {
+  const cardPaddingOnly = 48;
+  const aLittleExtra = 2;
+  const noImageDefault = cardPaddingOnly + aLittleExtra;
+  const imageBoxWidth = 190;
+  const imageSpacing = 10;
+  const inlineImageDefault = noImageDefault + imageBoxWidth + imageSpacing;
+
+  let maxWidth = 0;
+  if (!cardWidth) {
+    // default so initial result is 0 until actual width is known
+    maxWidth = inlineImage ? inlineImageDefault : noImageDefault;
+  } else {
+    maxWidth = cardWidth;
+  }
+  if (inlineImage) {
+    maxWidth -= inlineImageDefault;
+  } else {
+    maxWidth -= noImageDefault;
+  }
```

Comment:
Is this needed? When I open the inspector and remove the maxWidth style from the body and adjust the window size I didn't notice any issues.